### DISCUSSION TITLE
Vulkan example

### DISCUSSION THREAD

**Discussion by ib00** [AcademySoftwareFoundation/openvdb#1340] [Q&A]
Is there a Vulkan example of nanovdb (or cnanovdb)?

**Comment by gaida-exe** [comment]
I suggest that you look at the OpenVDB Viewer that uses NanoVDB/CNanoVDB.
https://github.com/AcademySoftwareFoundation/openvdb/tree/feature/nanovdb/nanovdb/nanovdb/cmd/viewer
